browser/main/lib/RcParser.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash'
+
+const path = require('path')
+const sander = require('sander')
+
+function parse (boostnotercPath) {
+  if (!sander.existsSync(boostnotercPath)) return
+  let config = JSON.parse(sander.readFileSync(boostnotercPath).toString())
+  return config
+}
+
+function exec (boostnotercPath) {
+  const config = this.parse(boostnotercPath)
+  if (config.execs === undefined) return
+  _.forEach(config.execs, (exec) => {
+    eval(exec)
```

Comment:
If you want to use plugin function, decide interface of the plugin and require it is better.
For example, ESLint plugin is so.

http://eslint.org/docs/developer-guide/working-with-plugins#processors-in-plugins

Please consider about it :)